lib/ncr_dispatcher.rb

```diff
@@ -0,0 +1,6 @@
+class NcrDispatcher < LinearDispatcher
```

Comment:
Argh, yeah, really hoping to avoid more NCR-specific logic. The important thing here is that the requesters don't want as many emails? It sounded like they had mixed opinions on it, so I am reluctant to add the logic to give them less information until they are actually using the system.